### PR TITLE
Feature - Copy all changes made to the docker honeypot fs to an overlay dir in session folder.

### PR DESCRIPTION
**Pull request by bang-uin**

### CODE REVIEW

**Comment by bang-uin** [comment]
This needs to be merged before i can add the pull request for container_reuse as it is based on fs_watch.

File: twisted_fix/internet/inotify.py [hunk 1]
```diff
@@ -0,0 +1,428 @@
+# -*- test-case-name: twisted.internet.test.test_inotify -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
```
Comment (tnich) [inline twisted_fix/internet/inotify.py]: Where did this come from?

Comment (tnich) [inline twisted_fix/internet/inotify.py]: Could you just create a class that inherits the actual twisted inotify, then overwrites the functions that you need changing?

**Comment by bang-uin** [comment]
twisted License file added :)
Sorry, forget that one

**Comment by tnich** [comment]
Looks good, I'll try and test it and merge later :smile:

**Comment by bang-uin** [comment]
BTW: This would also solve https://github.com/tnich/honssh/pull/70

**Pull request merged**